src/Symfony/Component/Console/Tests/Fixtures/application_2.md

```diff
@@ -178,6 +178,15 @@ The output format (txt, xml, json, or md)
 * Is multiple: no
 * Default: `'txt'`
 
+#### `--show-hidden`
+
+To show hidden commands
```

Comment:
Maybe `To include hidden commands`, it kinda implies we lose the other ones. Anyway this made doubt between `--show-hidden` and `--with-hidden`. Im fine either way, but perhaps be explicit in its description.